iib/workers/tasks/opm_operations.py

```diff
@@ -900,14 +900,14 @@ def verify_operators_exists(
     :param list(str) operator_packages: operator_package to check
     :param str overwrite_from_index_token: token used to access the image
     :return: packages_in_index, index_db_path
-    :rtype: (list, str)
+    :rtype: (set, str)
     """
     from iib.workers.tasks.build import terminate_process, get_bundle_json
     from iib.workers.tasks.iib_static_types import BundleImage
     from iib.workers.tasks.utils import run_cmd
     from iib.workers.tasks.utils import set_registry_token
 
-    packages_in_index: List[str] = []
+    packages_in_index: Set[str] = set()
```

Comment:
it would be nice to add a unit test for this

Reply:
unit tests were added, please check :) thank you!